podcasts/Enumerations.swift

```diff
@@ -399,15 +408,17 @@ extension PlayerAction: AnalyticsDescribable {
             return episode is UserEpisode ? "shelf_delete" : "shelf_archive"
         case .addBookmark:
             return "bookmarks-shelf-icon"
+        case .transcript:
+            return "show_notes"
         }
     }
 
     func canBePerformedOn(episode: BaseEpisode) -> Bool {
         switch self {
         case .starEpisode, .shareEpisode:
             return episode is Episode
-        case .addBookmark:
-            return isAvailable
+        case .transcript:
+            return FeatureFlag.transcripts.enabled
```

Comment:
Shouldn't we check that if the episode has transcripts available?

Reply:
Sorry, I missed adding this testing step.

We don't need it, if a transcript is not available the button will still be there but disabled.

Reply:
Thanks for clarifying! I see that is disabled when none is available!